cea/optimization/master/cost_model.py

```diff
@@ -211,15 +211,15 @@ def addCosts(indCombi, buildList, locator, dicoSupply, QUncoveredDesign, QUncove
         print pv.calc_Cinv_pv(PV_peak), "PV peak"
         
         SC_area = dicoSupply.SOLAR_PART_SC * solarFeat.SolarAreaSC
-        SCInvC = stc.calc_Cinv_SC(SC_area)
+        SCInvC = stc.calc_Cinv_SC(SC_area, gv)
```

Comment:
This is done to provide lifetime of solar collector, as it was hardcoded in the solar_collector.py. But you are right that all these need to be directed to another database containing cost parameters and removing 'gv' as such. It is being done in the branch 'i268-Technology-cost-database'.